### PR TITLE
[diffusion] add NVTX layerwise profiling support

### PR DESCRIPTION
**Pull request by wlc952**
## Summary
  
  Add `--enable-layerwise-nvtx-marker` CLI flag to the diffusion (multimodal_gen) 
  subsystem, mirroring the existing SRT feature. When enabled, PyTorch forward pre/post 
  hooks emit NVTX range markers for every sub-module in the pipeline (transformer/DiT, 
  text encoder, VAE), making all FFN, attention, and normalization layers visible in 
  nsys/ncu traces.

  ## Motivation

  SRT already supports layerwise NVTX profiling via `--enable-layerwise-nvtx-marker`. The
   diffusion subsystem had no equivalent, making it harder to profile model forward 
  passes with Nsight tools.
  
  ## Changes
  
  - `server_args.py`: add `enable_layerwise_nvtx_marker` dataclass field and 
  `--enable-layerwise-nvtx-marker` CLI argument
  - `gpu_worker.py`: import `PytHooks` from `sglang.srt.utils.nvtx_pytorch_hooks` and
  register hooks on each pipeline component module in `init_device_and_model()`
  
  ## How it works
  
  `PytHooks.register_hooks()` calls `torch.nn.Module.named_modules()` to recursively walk
   all sub-modules (attention, FFN, norms, etc.) and register `register_forward_pre_hook`
   / `register_forward_hook` pairs. Each pair emits `nvtx.range_push` / `nvtx.range_pop`
  with the module name and parameter metadata. No dependency changes required —
  `torch.cuda.nvtx` is bundled with PyTorch.

  ```
  # Usage
  sglang serve --model-path Wan-AI/Wan2.1-T2V-1.3B-Diffusers \
      --enable-layerwise-nvtx-marker

  # Then profile with nsys/ncu to see per-layer NVTX markers
  ```

  ## Test plan

  - [x] Launch a diffusion server with `--enable-layerwise-nvtx-marker` and verify no
  errors during model loading
  - [x] Run a generation request and capture an nsys profile — confirm per-layer NVTX
  ranges appear for transformer, text encoder, and VAE sub-modules
  - [x] Verify the flag defaults to `False` and has zero overhead when disabled

---
### CI States

Latest PR Test (Base): :x: [Run #26623069006](https://github.com/sgl-project/sglang/actions/runs/26623069006)
Latest PR Test (Extra): :x: [Run #26623068834](https://github.com/sgl-project/sglang/actions/runs/26623068834)

### CODE REVIEW

File: python/sglang/multimodal_gen/runtime/server_args.py [hunk 1]
```diff
@@ -1507,6 +1510,12 @@ def add_cli_args(parser: FlexibleArgumentParser) -> FlexibleArgumentParser:
             default=ServerArgs.otlp_traces_endpoint,
             help="OTLP collector endpoint when --enable-trace is set. Format: <host>:<port>",
         )
+        parser.add_argument(
+            "--enable-layerwise-nvtx-marker",
```
Comment (mickqian) [inline python/sglang/multimodal_gen/runtime/server_args.py:1514]: we already have this

**Review by mickqian — CHANGES_REQUESTED** [review]
